manifests/v/vim/vim/9.0.0049/vim.vim.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: vim.vim
+PackageVersion: 9.0.0049
```

Comment:
```suggestion
PackageVersion: 9.0.49
```

Reply:
> @Trenly, not sure I understand. I assume this is about what is stored in the installer as version info?

The PackageVersion must match what is shown in the `A`dd and `R`emove `P`rograms section of control panel when the application is installed. This is because the `DisplayVersion` registry key is matched with `PackageVersion` to match installations to packages in the repository



![image](https://user-images.githubusercontent.com/12611259/178281512-5149eb39-4be9-487e-a36b-f2c8353cc558.png)

Reply:
I have to investigate

Reply:
tracking this issue here: https://github.com/vim/vim-win32-installer/issues/276 I am not sure how fast we will be in resolving this.

Reply:
this should be fixed now